index.js

```diff
@@ -1,19 +1,4 @@
 'use strict';
-module.exports = function (pth, cb) {
-	if (String(pth).indexOf('\u0000') !== -1) {
-		var err = new Error('Path must be a string without null bytes.');
-		err.code = 'ENOENT';
-
-		if (typeof cb !== 'function') {
-			throw err;
-		}
-
-		process.nextTick(function () {
-			cb(err);
-		});
-
-		return false;
-	}
-
-	return true;
-}
+module.exports = function (pth) {
+	return String(pth).indexOf('\u0000') > -1;
```

Comment:
Wouldn't it make more sense if it still threw? The whole point of this module is not having to redefine the error message and code.